master/getting-started/bare-metal/bare-metal.md

```diff
@@ -387,8 +387,8 @@ policy, Calico has a failsafe mechanism that keeps various pinholes open
 in the firewall.
 
 By default, Calico keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as outbound communication to ports
-2379, 2380, 4001 and 7001, which allows access to etcd's default ports.
+which allows access to ssh; as well as inbound and outbound communication to
+ports 2379, 2380, which allows access to etcd's default ports.
```

Comment:
Worth mentioning the 6xxx ports for etcd as deployed by <installer>?

Reply:
how about "ports 2379, 2380, 6666 and 6667, which allows access to etcd's default ports and etcd's ports deployed as kubernetes pods by calico manifests."

Reply:
How about "as well as inbound and outbound communications to ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports as deployed by Calico's self-hosted Kubernetes manifests)."

Reply:
Agreed.